src/i_oalsound.c

```diff
@@ -238,8 +235,7 @@ static void SetResampler(ALuint *sources)
     }
     if (i == num_resamplers)
     {
-        I_Printf(VB_WARNING, " Failed to find resampler: '%s'.",
-                 resampler_name);
+        I_Printf(VB_WARNING, " Failed to find resampler: '%s'.", snd_resampler);
```

Comment:
Is it possible to add "Available resampler names are: ..."?

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1637/commits/e3f3d6e8a9c3eed21925c05c93fc0f49709104ab

![Untitled](https://github.com/fabiangreffrath/woof/assets/56656010/d598a85f-7970-4efc-9c7b-96dde2b6a23e)

Reply:
Thank you! Did this mean we would have to pass strings such as "23rd order Sinc (fast)" verbatim?

Reply:
Yes, I don't see a way to query the short names unfortunately (`fast_bsinc24`, etc.).